src/components/common/ShareStructure/ShareStructures.vue

```diff
@@ -78,18 +79,19 @@ export default class ShareStructures extends Mixins(CommonMixin) {
     return this.getEntitySnapshot?.shareStructure
   }
 
-  /** Is true if changes to share structure rights will require a resolution date. */
+  /** True if changes to share structure rights will require a resolution date. */
   get resolutionsRequired (): boolean {
     return (this.getNewResolutionDates.length === 0) && this.isAlterationFiling
   }
 
   /** Keep the store in sync with components state of validity. */
-  @Watch('invalidMinimumShareClass')
-  private setShareStructureValidity (isEditing: boolean): void {
```

Comment:
This method was called directly when this component received a "emitEditingShareStructure" event (in which case `isEditing` is the event value), and it was called when "invalidMinimumShareClass" changes (in which case `isEditing` is the value of invalidMinimumShareClass -- incorrect!!).

I fixed this by adding a watcher and ignoring the watcher value.